docs/config.adoc

```diff
@@ -781,9 +781,20 @@ The mouse movement actions are:
 * `movemouse-left`
 * `movemouse-right`
 
-Similarly to the mouse wheel, these actions accept two number strings. The first
-is the interval (unit: ms) between movement actions and the second number is the
-distance (unit: pixels) of each movement.
+All of these actions accept either three or five number strings, depending on the
+value of the first number. The first number is determines if acceleration is on 
```

Comment:
While for the internal code it makes sense to unify the accelerating/non-accelerating mouse movement, I think that for the configuration language, it makes more sense to keep them as separate actions as it was before.

The first parameter changing behaviour depending on being a 1 or a 0 seems like a non-obvious way to have differing behaviour, particularly when skimming a configuration file.

Reply:
That makes sense, hopefully now I've done it how you wanted it to be. 👍
Sorry that it took quite the bit of going back and fourth.

Reply:
All good, thanks for doing the work!